capstone-backend/src/main/java/com/google/sps/data/Club.java

```diff
@@ -35,15 +29,23 @@ public Club(long id, String name, String description, String announcement, Colle
         this.gbookID = gbookID;
   }
 
-  public Club(long id, String name, String description, String announcement, long ownerID, String gbookID){
-    this(id, name, description, announcement, new ArrayList<String>(), ownerID, new HashSet<Long>(), new HashSet<Long>(), gbookID);
+  public Club(String id, String name, String description, String ownerID, String gbookID){
+    this(id, name, description, "", new ArrayList<String>(), ownerID, new ArrayList<String>(), new ArrayList<String>(), gbookID);
+  }
+
+  public Club(String id, String name, String ownerID, String gbookID) {
+    this(id, name, "", ownerID, gbookID);
+  }
+
+  public Club(String id, String name, String ownerID) {
+    this(id, name, "", ownerID, "");
   }
 
-  public Club(long id, String name, String description, long ownerID, String gbookID) {
-    this(id, name, description, "", ownerID, gbookID);
+  public Club() {
```

Comment:
I know yesterday we talked about this a bit offline - but I was curious what the use of the default constructor is for Firestore?

Reply:
I believe it lets Firestore cast the objects it retrieves (cast to type Club in this case). I believe you can just use the default Java constructor too.